src/compiler/compile.test.ts

```diff
@@ -55,24 +48,17 @@ describe('Create correct mapping table between Gosling track IDs and HiGlass vie
                 }
             ]
         };
-        compile(
-            spec,
-            (h, s, g, t, table) => {
-                expect(table).toMatchInlineSnapshot(`
-                  {
-                    "o1": "o1",
-                    "o2": "o1",
-                    "o3": "o1",
-                    "s1": "s1",
-                    "s2": "s2",
-                    "s3": "s3",
-                  }
-                `);
-            },
-            [],
-            getTheme(),
-            {}
-        );
+        const { idTable } = compile(spec, [], getTheme(), {});
+        expect(idTable).toMatchInlineSnapshot(`
+          {
+            "overlay-0060b963": "overlay-0060b963",
+            "overlay-01333c96": "overlay-0060b963",
+            "overlay-c09472ed": "overlay-0060b963",
+            "s1": "s1",
+            "s2": "s2",
+            "s3": "s3",
```

Comment:
Still 4 test fails related in `compile.test.ts`. I think there was some change on how easy overlaid tracks are assigned with ID.